simulation-scripts/perturb.sh

```diff
@@ -513,12 +515,12 @@ run_file_on_host() {
   #shellcheck disable=SC2094
   (
     touch "${TMP_DIR}/perturb-script-file-${HOST}.log"
+    get_file_to_run "${FILE}" >> "${TMP_DIR}/perturb-script-file-${HOST}.log"
     exec 19>>"${TMP_DIR}/perturb-script-file-${HOST}.log"
     BASH_XTRACEFD=19
     set -x
-    get_file_to_run "${FILE}" >> "${TMP_DIR}/perturb-script-file-${HOST}.log" 2>&1
```

Comment:
Maybe we should rename this function to something like `cat_script_to_run` or `output_script_to_run`?  I had to look it up to figure out what it was doing

Reply:
Done